source/crud/tests/db/db-aggregate.yml

```diff
@@ -0,0 +1,58 @@
+data: []
+
+tests:
+    -
+        description: "Aggregate with $currentOp"
+        operation:
+            name: db-aggregate
+            database_name: admin
+            arguments:
+                pipeline:
+                  - $currentOp: {allUsers: true, idleConnections: true}
```

Comment:
Would `false` for both of these be more robust on the off chance other operations are running on the server? Given that the expected output is only this database-level aggregation, it seems reasonable to ignore other things if possible.

That might also be more portable, since `allUsers: true` requires an `inprog` privilege. I assume most drivers would have that permission during testing, but it's still an extra loose end.